bootstrap/README.md

```diff
@@ -0,0 +1,59 @@
+# Bootstrap script
+This script is used to set up an org with the input, output schema and input, output datasets 
+required for the recipe workflow.
+
+# Workflow
+Following are the steps the script runs through to setup the input and output datasets in the org:
+1. Get the access token to access the APIs
+2. Get the tenantID of the org
+3. Create Retail Sales Class
+4. Create Input Retail Sales Mixin with the class url
+5. Create Input Retail Sales Schema with the class url and the mixin url
+6. Create Input Dataset with the schema url
+7. Create Batch with the dataset id
+8. Replace the tenant id in the json 
+9. Upload the datafile (with the tenantID from above)
+10. Close the batch
+11. Create Output Retail Sales Mixin with the class url from step 3
+12. Create Output Retail Sales Schema with the class url and the mixin url
+13. Create Output Dataset with the schema
+
+# Setup
+We would need an access token to access the APIS. This can be obtained from either an Adobe IO integration or from the 
```

Comment:
Perhaps less informal here: `An access token is needed to access the APIs` (note `APIs` vs. `APIS`)